jakarta.ls/src/main/resources/transactions.json

```diff
@@ -1,3 +1,53 @@
 {
-
+	"Initialize a UserTransaction object via Injection": {
```

Comment:
Looks like there may be tabs in this document, to keep consistent with https://github.com/eclipse/lsp4jakarta/pull/259 use spaces instead of tabs

Reply:
Fixed! Didn't realize, sorry!